compose-builder/Makefile

```
@@ -974,6 +985,15 @@ define COMPOSE_DOWN
 		down $1
 endef
 
+ifeq (zero-trust, $(filter zero-trust,$(ARGS)))
+	# overrides for zero trust from the base files
+	COMPOSE_FILES:=$(COMPOSE_FILES) -f docker-compose-base-zero-trust.yml
```

Rename file since it is not a base, but an addition...
```suggestion
	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-security-zero-trust.yml
```

---

all set